buster/formatters/questions.py

```diff
@@ -0,0 +1,22 @@
+import logging
+
+from buster.completers import ChatGPTCompleter
+
+
+class QuestionReformulator:
+    def __init__(self):
+        completion_kwargs = {
+            "model": "gpt-3.5-turbo",
+            "stream": False,
+            "temperature": 0,
+        }
+        self.completer = ChatGPTCompleter(completion_kwargs=completion_kwargs)
+        self.system_prompt = """
```

Comment:
should go into the config file

Reply:
ill leave a default prompt but add ability to override in the cfg file

Reply:
Also: move this to a `llm_utils` folder